Detectors/ITSMFT/common/simulation/src/AlpideSimResponse.cxx

```diff
@@ -24,9 +24,9 @@ using namespace o2::ITSMFT;
 using namespace std;
 
 ClassImp(o2::ITSMFT::AlpideSimResponse)
-ClassImp(o2::ITSMFT::AlpideRespSimMat)
+  ClassImp(o2::ITSMFT::AlpideRespSimMat)
```

Comment:
this looks weird. Could you add a `;` at the end and reformat, please? Can be merged directly thereafter.